go/exocom/src/client_registry/client_registry.go

```diff
@@ -0,0 +1,77 @@
+package clientRegistry
+
+import "encoding/json"
+
+// Client is the combination of a client name, service type and internal namespace
+type Client struct {
+	ClientName        string
```

Comment:
I believe its role in the service routes but was named clientName internally. Will update in a separate PR